src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,29 @@
+"""Build go-spdk bindings C tests"""
+from __future__ import print_function
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement
(pylint-reimported) Reimport 'print_function' (imported line 3)

Reply:
It’s because it’s a Sconscript.  Checkscript adds a custom preamble before running checkpatch.

I’d have to look at the wrap script (see check_script.py in scons_local) to figure out if there is something we can do differently.  I suspect it’s getting confused by the #python comment which is unnecessary for a Sconscript

Reply:
addressed in follow-up: https://github.com/daos-stack/daos/pull/1505